src/platform/datapath_winuser.c

```diff
@@ -706,27 +700,8 @@ CxPlatDataPathQuerySockoptSupport(
         Datapath->Features |= CXPLAT_DATAPATH_FEATURE_RECV_COALESCING;
     }
 }
-    //
-    // TODO: This "TTL_FEATURE check" code works, and mirrors the approach for Kernel mode.
-    //       However, it is considered a "hack" and we should determine whether or not
-    //       the current release story fits this current workaround.
-    //
-    HMODULE NtDllHandle = LoadLibraryA("ntdll.dll");
-    if (NtDllHandle) {
-        FuncRtlGetVersion VersionFunc = (FuncRtlGetVersion)GetProcAddress(NtDllHandle, "RtlGetVersion");
-        if (VersionFunc) {
-            RTL_OSVERSIONINFOW VersionInfo = {0};
-            VersionInfo.dwOSVersionInfoSize = sizeof(VersionInfo);
-            if ((*VersionFunc)(&VersionInfo) == 0) {
-                //
-                // Some USO/URO bug blocks TTL feature support on Windows Server 2022.
```

Comment:
I think this comment (or an improved version of it) should be kept.
Otherwise, 20348 is just a magic number and soon, nobody will know why TTL is disabled on that build.

Reply:
Agreed, I will add back those comments in the next iteration.